examples/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
I expect more wrappers to be deleted from the sub-projects, did you forget to do that?